### DISCUSSION TITLE
Permanent Keep Speed und Automatische Gruppenschaltung

### DISCUSSION THREAD

**Discussion by iSinterra** [Mogli12/KeyboardSteer#835] [Q&A]
Hallo, erstmal vielen Dank für diesen tollen Mod. Hab leider ein kleines Problem. Weiß auch nicht, ob das so gewollt ist. Aber seit dem letzten Update (denke ich), werden mit aktiviertem "Permanent Keep Speed" keine Gruppen mehr automatisch hoch geschaltet. Gänge ja, Gruppen nein. Getestet mit dem Landini.

**Comment by Mogli12** [comment]
VCA ändert nichts an der automatischen Schaltung.

**Comment by iSinterra** [comment]
https://youtu.be/lFPriG0nBnU

Was soll ich sagen, er hängt in der ersten Gruppe. Beim 2. Versuch hab ich eine Gruppe manuell hoch geschaltet, dann schaltet er auch die restlichen durch. Nach dem Anhalten, das Gleiche wieder.

**Comment by Mogli12** [comment]
Hast Du schon mal probiert, mit dem Tempomaten zu fahren?

**Reply by iSinterra** [reply]
Ja, der funktioniert ganz normal.

**Comment by Mogli12** [comment]
Ich kann es nur nochmal schreiben: VCA ändert gar nichts an der automatischen Schaltung selber. Wenn sich FS22 überlegt, den Gang nicht zu wechseln, weil als Geschwindigkeitslimit 10,9 km/h eingestellt ist, dann ist das leider so.

**Comment by iSinterra** [comment]
Ja keine Ahnung was es ist. Ist jedenfalls nicht nur ein Problem mit dem Schaltgetriebe. Hab ich, zum Beispiel, ein Pflug hinter dem Xerion (CVT Getriebe) und fahre mit aktiviertem KeepSpeed los. Hängt er bei 3 kmh und wird nicht schneller. Schalte ich KeepSpeed aus, 12 kmh ohne Probleme. Das gleiche, wenn man Bergauf fährt. Mit KS geht die Geschwindigkeit runter, fast bis zum Stillstand. Schalte ich es aus, zieht er den Berg hoch, ohne Probleme. Was mich wundert, wenn ich Cruise Control auf 12 kmh einstelle und KS mal wieder bei 3kmh hängt. Dann die Cruise Control Geschwindigkeit an KS übergebe, zieht er sofort los auf 12. Xerion hier nur als Beispiel, ist mit jedem anderen Gefährt das Gleiche. Hab es eigentlich immer gerne genutzt, zum Beispiel im Vorgewende oder als Vario Ersatz um mit dem Joystick die Geschwindigkeit zu regeln. Ist zurzeit leider aber nur nervig. Ist vielleicht wieder nur ein Server Problem, dass da was nicht richtig ausgelesen wird. Singleplayer hab ich nicht getestet.

**Reply by iSinterra** [reply]
Würde vielleicht schon reichen, wenn man ne Option hätte die Gänge automatisch schalten zu lassen, aber die Gruppen nur manuell. Keine Ahnung ob das machbar ist.
